.github/workflows/test-nodejs-nightly.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ 'macos-14', 'macos-14-large', 'ubuntu-24.04', 'ubuntu-24.04-arm', 'windows-11-arm', 'windows-2022' ]
+        os: [ 'macos-15', 'ubuntu-24.04', 'ubuntu-24.04-arm', 'windows-11-arm', 'windows-2022' ]
```

Comment:
Is there a specific reason not to test on intel macos anymore?

Reply:
[Here's the current list.](https://github.com/actions/runner-images) Looks like `macos-15-large` is intel.